### ISSUE TITLE
Dead Authors' Series Continued by Other Authors

### ISSUE THREAD

**Issue by adamgoldberg** [DobyTang/LazyLibrarian#805]
Suppose I want the book "Full Force and Effect", which was written by Mark Greaney, but is really "Tom Clancy's Full Force and Effect by Mark Greaney"... The way the data gets downloaded from Goodreads, it searches for "Mark Greaney Full Force and Effect", and doesn't find it.  But if I manually edit the book to make the author Tom Clancy, searches find it, and it downloads correctly (but, of course, it gets renamed "Full Force and Effect - Tom Clancy" which isn't quite right, probably).

Is there a way to make this work in the existing code (with less manual processing)?  Or ...??

**Comment by philborman** [comment]
Difficult. Goodreads correctly says the author is Mark Greaney,  some of the uploaders (torrents/nzb) put  Tom Clancy in the title and don't mention Mark Greaney. Goodreads gives us a table of book authors with their "role" which we can use to filter out narrators, translators, cover artists and the like, but that table isn't provided for all books.

The book is part of the "Jack Ryan Universe" series, so using the series tab will pull the various authors  together once you've got the book, but there isn't an easy way of automating the author change at the moment.  There is a similar problem with the "Jason Bourne" series , Robert Ludlum and then Eric van Lustbader, or the "James Bond" books. Suggestions welcome ;-)

**Comment by adamgoldberg** [comment]
Is there a way to edit a book to say "search for this by title only" (non-manually)?  Maybe that'd be useful?

**Comment by philborman** [comment]
There used to be an option to do that, but it gave far too many false
matches and downloaded the wrong books so it was removed. It's surprising
how many book titles are the same but different authors. An obvious one is
"My Autobiography", but even "Full force and effect" gives 6 matches, and
only one is the right book. I think that's best left to the user to choose
the right match.

**Comment by adamgoldberg** [comment]
I guess I'm suggesting something like when you go to mark a book as 'wanted', there might be a drop-down box with search by title/author, title, author (where title/author is the default).  Or something like that.

**Comment by philborman** [comment]
That's fairly easy to add. I'm looking at something similar for audiobooks,
do you want the ebook or the audiobook or both. Not sure what to do with
automatic downloads though. You can configure lazylibrarian to watch for
new books by an author and download them automatically. You probably don't
want "title only" search on all books. Actually, how do you know you want
"title only" unless you've tried "author and title" first?

**Comment by adamgoldberg** [comment]
Seems like the Clancy/Greaney problem calls for a title-only search ... rare, but...

**Comment by philborman** [comment]
We used to have an extra search in the "booksearch" process, if you couldn't find the book by author_and_title, retry with just title, but as I said earlier it caused too many false matches so was removed. I think the Clancy/Greaney problem is a good example, if we search for "Mark Greaney Full Force and Effect" we don't get any results, if we search for just "Full Force and Effect" we find results but the author is Tom Clancy, so if we scan all the results for "Mark Greaney" we still don't get a match. How is lazylibrarian supposed to know it's the right book? I think this has to be done manually, unless anyone has a better idea of course...   I will close this for now.
